### DISCUSSION TITLE
Versioned Migrations

### DISCUSSION THREAD

**Discussion by liemle3893** [ariga/atlas#565] [Q&A] [answered]
I see that we have plan for **Versioned Migrations**. Just want to ask is there any roadmap, status about this feature?

**Comment by masseelch** [comment, marked as answer]
Good Morning @liemle3893 🌅,

I currently am working on that feature. I plan to have it available as a package in next week's release. 

It should be part if the cli on week later. 

The very first part made it into master [yesterday](https://github.com/ariga/atlas/pull/556) and the rest will follow in separate PRs. Since we develop Atlas in Open Source you are welcomed to give feedback or share your opinions at any time. 😊

**Reply by liemle3893** [reply]
Love to hear that. I'll follow the progress and join you guys if I can help.
